### PR TITLE
[GAL-3616] Update NFT Detail Page [mobile]

### PR DESCRIPTION
**Pull request by jakzaizzat**
**Changes**
- User able to post in NFT Detail page
- Update NFT detail page with new design

**Notes**
Depend on this PR: https://github.com/gallery-so/gallery/pull/1682

**Demo**

**NFT Detail page of own token**

https://github.com/gallery-so/gallery/assets/4480258/9f10cd7c-4717-4fa3-b52d-73a262cfe8a9

**NFT Detail page of other users token**

https://github.com/gallery-so/gallery/assets/4480258/a1f01905-b7ac-44aa-bc4a-70cf78ea4fa5

**NFT Detail Ethereum**
CleanShot 2023-07-13 at 15 01 06@2x

**NFT Detail Tezos**
CleanShot 2023-07-13 at 15 02 13@2x

**NFT Detail POAP**
CleanShot 2023-07-13 at 15 02 59@2x

### CODE REVIEW

**Comment by kaitoo1** [comment]
should this be a PR into the `jakz/mobile-composer` branch? might make seeing the unique changes in the diff easier 👀

**Comment by jakzaizzat** [comment]
Sounds good @kaitoo1! Changed the base branch to `jakz/mobile-composer`

File: apps/mobile/src/icons/CirclePostIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import Svg, { Path, SvgProps } from 'react-native-svg';
```
Comment (kaitoo1) [inline apps/mobile/src/icons/CirclePostIcon.tsx:1]: I think some of the screens in the figma used this old Circle icon by mistake, we should use the same icon we have in the footer (square border around the plus)

File: apps/mobile/src/screens/NftDetailScreen/NftAdditionalDetailsTezos.tsx [hunk 2]
```diff
@@ -60,19 +62,19 @@ export function NftAdditionalDetailsTezos({
...
+            <DetailLabelText>CREATED BY</DetailLabelText>
 
-          {/* TODO(Terence) When the contract screen is ready, setup the onPress here */}
-          <LinkableAddress
-            chainAddressRef={token.contract.creatorAddress}
-            type="NFT Detail Creator Address"
-          />
-        </DetailSection>
-      )}
+            {/* TODO(Terence) When the contract screen is ready, setup the onPress here */}
```
Comment (kaitoo1) [inline apps/mobile/src/screens/NftDetailScreen/NftAdditionalDetailsTezos.tsx:70]: is this comment still valid? looks like `LinkableAddress` might handle pressing on the address

**Review by kaitoo1 — APPROVED** [review]

**Comment by kaitoo1** [comment]
> Sounds good @kaitoo1! Changed the base branch to `jakz/mobile-composer`

thanks @jakzaizzat ! made reviewing a lot easier 😁

**Pull request merged**